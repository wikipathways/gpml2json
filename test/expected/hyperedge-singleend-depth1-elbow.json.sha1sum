8eb0005dbe6f1ac2cdd2e95a9132a28c8fd613c7 ?-
