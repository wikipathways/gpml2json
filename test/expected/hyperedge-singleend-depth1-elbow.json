{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["idaf905386","id28382a1c","faa53"],"drawAs":"rect","gpmlElementName":"Pathway","height":193.20987654320973,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":193.78600823045275,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130715","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/2774a3d88fe999a831f545d067c3103e4e9326dc644a278f7f57fd11197ba356"},"entityMap":{"faa53":{"id":"faa53","textContent":"GeneProduct","wpType":"GeneProduct","x":30.71829405162859,"y":143.20987654320973,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"idaf905386":{"id":"idaf905386","drawAs":"ElbowLine","borderWidth":1,"zIndex":12288,"points":[{"x":30.452674897119397,"y":66.6666666666666,"orientation":[-1,0]},{"x":10.452674897119397,"y":81.06060606060603},{"x":99.31724653946877,"y":95.45454545454545},{"x":188.18181818181816,"y":81.06060606060602},{"x":163.78600823045275,"y":66.66666666666659,"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","burrs":["e0874"]},"e0874":{"id":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id28382a1c":{"id":"id28382a1c","drawAs":"ElbowLine","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"faa53","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":70.71829405162859,"y":143.20987654320973,"orientation":[0,-1]},{"x":94.74946077140926,"y":123.20987654320973},{"isAttachedTo":"e0874","attachmentDisplay":{"position":[0.5617283950617287,0],"offset":[],"relativeOffset":[0,0]},"x":118.78062749118993,"y":95.45454545454545,"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["faa53","idaf905386"]}}}
